README.md

```diff
@@ -26,6 +26,12 @@ Create an HPI file to install in Jenkins (HPI file will be in `target/slack.hpi`
 
     mvn package
 
+If you are having trouble getting mvn to package due to a test failure, please add this to your pom.xml
```

Comment:
Not sure this belongs in the pom of the project. Also, you can skip tests (or even skip compiling tests) from your mvn command:  

```
mvn install -DskipTests
mvn install -Dmaven.test.skip=true
```